src/app/admin/page.tsx

```diff
@@ -1,5 +1,235 @@
+        >
+          <h1
+            id="src-app-admin-page-mainContainer-gridContainer-top5Products-title"
+            className="text-2xl font-bold mt-2"
+          >
+            Top 5 most selled products
+          </h1>
+          <BasicLineChart
+            marginTop={120}
+            width={400}
+            xAxis={topFiveProd.map((e, i) => ({
+              data: topFiveProd[i].dailySales.map((e: any) =>
+                parseFloat(e.dayNumber)
+              ),
+            }))}
+            series={topFiveProd.map((e, i) => ({
+              data: topFiveProd[i].dailySales.map((e: any) => e.totalSales),
+              label: e.name,
+            }))}
+          />
+        </div>
+      </div>
+      <div
+        id="src-app-admin-page-mainContainer-gridContainer-ordersAndSales"
+        className="w-full p-4 gap-5 items-center grid grid-cols-1 grid-rows-2 md:grid-cols-2 md:grid-rows-1"
+      >
+        <div
+          id="src-app-admin-page-mainContainer-gridContainer-ordersAndSales-newOrders30d"
+          className="flex flex-col gap:1 justify-start rounded-xl items-center bg-white shadow-xl overflow-x-auto"
+        >
+          <h1
+            id="src-app-admin-page-mainContainer-gridContainer-ordersAndSales-newOrders30d-title"
+            className="text-2xl font-bold mt-2"
+          >
+            New orders 30D
+          </h1>
+          <BasicLineChart
+            marginTop={50}
+            width={400}
+            xAxis={[
+              { data: orders30d.map((entry) => entry.dayNumber) },
+              { data: notPaidOrders30d.map((entry) => entry.dayNumber) },
+              { data: paidOrders30d.map((entry) => entry.dayNumber) },
+            ]}
+            series={[
+              {
+                data: orders30d.map((entry) => entry.count),
+                color: "#A0A0A0",
+                label: "Orders",
+                area: true,
+              },
+              {
+                data: notPaidOrders30d.map((entry) => entry.count),
+                color: "#DE3163",
+                label: "Not paid orders",
+                area: true,
+              },
+              {
+                data: paidOrders30d.map((entry) => entry.count),
+                label: "Paid orders",
+                color: "#00CC77",
+                area: true,
+              },
+            ]}
+          />
+        </div>
+        <div
+          id="src-app-admin-page-mainContainer-gridContainer-ordersAndSales-totalSales30d"
+          className="flex flex-col gap:1 justify-start rounded-xl items-center bg-white shadow-xl"
+        >
+          <h1
+            id="src-app-admin-page-mainContainer-gridContainer-ordersAndSales-totalSales30d-title"
+            className="text-2xl font-bold mt-2"
+          >
+            Total sales 30D
+          </h1>
+          <BasicLineChart
+            marginTop={50}
+            marginLeft={40}
+            xAxis={[{ data: totalSales30d.map((entry) => entry.dayNumber) }]}
```

Comment:
We need to add a logic here to display the whole month, not only the days that have value.